freqtrade/constants.py

```diff
@@ -46,7 +46,7 @@
 # it has wide consequences for stored trades files
 DEFAULT_TRADES_COLUMNS = ['timestamp', 'id', 'type', 'side', 'price', 'amount', 'cost']
 TRADING_MODES = ['spot', 'margin', 'futures']
-MARGIN_MODES = ['cross', 'isolated']
+MARGIN_MODES = ['cross', 'isolated', '']
```

Comment:
I think you removed this, it looks like the `test_commands` tests fail without it

Reply:
what i changed was the variable name to MARGIN_MODES - but i didn't touch the content ... 

![image](https://user-images.githubusercontent.com/5024695/152409253-1782f1ec-004a-4d2b-8e68-b3786cad5dd7.png)

Reply:
I forgot this change hadn't be committed yet, I changed it on the `todos` branch, sorry